composer.json

```diff
@@ -3,25 +3,32 @@
     "description": "eLife Sciences logging SDK",
     "type": "library",
     "license": "MIT",
-    "require": {
-        "monolog/monolog": "^1.22"
-    },
-    "require-dev": {
-        "phpunit/phpunit": "^5.6"
-    },
-    "suggest": {
-        "ext/newrelic": "Enables the Monitoring class"
-    },
     "autoload": {
         "psr-4": {
             "eLife\\Logging\\": "./src"
         }
     },
     "autoload-dev": {
         "psr-4": {
-            "tests\\eLife\\Logging\\": "./tests"
+            "test\\eLife\\Logging\\": "./test"
+        }
+    },
+    "require": {
+        "monolog/monolog": "^1.22",
+        "psr/log": "^1.0"
+    },
+    "require-dev": {
+        "phpunit/phpunit": "^5.7 || ^6.0"
```

Comment:
since it's require-dev no reason not to require `^6.0` only?

Reply:
Just to avoid any constraint problems in the prefer-lowest tests. (Current dependencies won't cause a problem, but any more in the future might.)